Tests/FluentAssertions.Json.Shared.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -457,6 +457,70 @@ public void When_jtoken_does_have_value_NotHaveValue_should_fail()
 
         #endregion (Not)HaveValue
 
+        #region (Not)MatchRegex
+
+        [Fact]
+        public void When_jtoken_matches_regex_pattern_MatchRegex_should_succeed()
```

Comment:
🙃  The convention is to use _when_....._it should_
❌  I prefer not to use literal terms of code elements in the title of a test case. See also point 12 of https://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html